addons/mail/static/src/model/model_field.js

```diff
@@ -788,7 +793,7 @@ class ModelField {
             this.env.modelManager._update(
                 otherRecord,
                 { [this.inverse]: [['unlink', record]] },
-                { hasToUpdateInverse: false }
+                { hasToUpdateInverse: false, allowWriteReadonly: true }
```

Comment:
I think you are missing more cases in this file (double check every time we write ´hasToUpdateInverse: false´). There must be 4 cases of inverse relation: link/unlink & x2o/x2m